### PR TITLE
Remove mention of *.class parameters from conventions

### PR DESCRIPTION
**Pull request by jvasseur**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |

### CODE REVIEW

**Comment by stof** [comment]
:+1:

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -167,9 +167,6 @@ Service Naming Conventions
 
 * A group name uses the underscore notation;
```
Comment (xabbuh) [inline contributing/code/standards.rst]: Can you please replace the trailing semicolon with a dot?

Comment (jvasseur) [inline contributing/code/standards.rst]: Done.

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thanks for keeping the docs in sync with new decisions in Symfony world @jvasseur!

**Pull request merged**